### PR TITLE
Milestone: Add "join table" for visits

### PR DESCRIPTION
**Pull request by Kashif-Saleem-Ghuman**
In this milestone 💪 

- Made different types of relationships between tables.
- Prepared complex queries that answer analytical questions.
- Used primary key & foreign key mechanisms for joining tables.
- Screenshots
- Who was the last animal seen by William Tatcher?

-How many different animals did Stephanie Mendez see?

-List all vets and their specialties, including vets with no specialties.

-List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.

-What animal has the most visits to vets?

-Who was Maisy Smith's first visit?
marry_smith_first_visit
-Details for most recent visit: animal information, vet information, and date of visit.

-How many visits were with a vet that did not specialize in that animal's species?

-What specialty should Maisy Smith consider getting? Look for the species she gets the most.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -111,3 +111,67 @@ LEFT JOIN animals ON owners.id = animals.owner_id
 GROUP BY owners.id
 ORDER BY count DESC
 LIMIT 1;
+
```
Comment (t-yanick) [inline queries.sql:114]: [ OPTIONAL] 
For better documentation, I suggest you put the query questions as comments at the top of each query block

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @Kashif-Saleem-Ghuman , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  SQL commands all in the correct files
:heavy_check_mark: Descriptive PR with requested screenshots

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] Great job working on this milestone and adding the queries screenshots, but it seems you are missing the query results of `Who was Maisy Smith's first visit?`
There are 9 queries in the project requirements, but you have 8 screenshots.
I will suggest you put the query questions just before the screenshots for better documentation.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Kashif-Saleem-Ghuman 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done on making the required changes from the previous reviewer

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kashif-Saleem-Ghuman** [comment]
@t-yanick Thank you for reviewing my PR and providing valuable feedback

**Comment by Kashif-Saleem-Ghuman** [comment]
@MahmoudBakr23 Thank you for approving my PR :)

**Pull request merged**